### PR TITLE
[release-0.49] Bump virtlogd memory requirement to 18Mi

### PR DESCRIPTION
**Pull request by fossedihelm**
Signed-off-by: fossedihelm <ffossemo@redhat.com>

**What this PR does / why we need it**:
This is a partial manual cherry-pick of https://github.com/kubevirt/kubevirt/pull/8247

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```
/cc @acardace

### CODE REVIEW

**Review by acardace — APPROVED** [review]

File: tests/vmi_configuration_test.go [hunk 1]
```diff
@@ -2963,7 +2963,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 	})
 	Context("virt-launcher processes memory usage", func() {
-		It("should be lower than allocated size", func() {
+		FIt("should be lower than allocated size", func() {
```
Comment (stu-gott) [inline tests/vmi_configuration_test.go:2966]: This line has a focus. This will break the test suite.

**Review by stu-gott — CHANGES_REQUESTED** [review]
One small change is needed.

**Comment by stu-gott** [comment]
/hold cancel

Since the LGTM was removed, the hold is not needed.

File: tests/vmi_configuration_test.go (hunk 1 above)
Comment (xpivarc) [reply tests/vmi_configuration_test.go:2966]: This will only break local development.

Comment (fossedihelm) [reply tests/vmi_configuration_test.go:2966]: Thanks both <3

**Pull request merged**